crypto/cryptlib.c

```diff
@@ -167,22 +167,6 @@ void OPENSSL_showfatal(const char *fmta, ...)
     va_list ap;
     TCHAR buf[256];
     const TCHAR *fmt;
-# ifdef STD_ERROR_HANDLE        /* what a dirty trick! */
```

Comment:
The whole section is _WIN32, it shouldn't be surprise that it Win thing... As for commentary. "We first want to check if it's console application, in which case error message would be printed on standard error. However Windows CE doesn't have concept of console application. We could guard the check with #ifdef _WIN32_WCE, but #ifdef STD_ERROR_HANDLE does the trick just as good."